src/app/visual/country-field/country-field-visual.component.html

```diff
@@ -0,0 +1,63 @@
+<div
+  style="padding: 10px"
+>
+  <h2
+    class="sky-section-heading"
+  >
+    Standard country field
+  </h2>
+
+
+  <div
+    id="screenshot-country-field-empty"
+    style="width: 50%; height: 100px; margin: 10px; padding: 10px;"
+  >
+    <form>
+      <sky-country-field
+        [(ngModel)]="countryData"
```

Comment:
2-way bound properties should come last.

Reply:
Done